README.md

```diff
@@ -64,3 +64,18 @@ Not supported
 [listTopics()](http://kafka.apache.org/21/javadoc/org/apache/kafka/clients/admin/AdminClient.html#listTopics--) returns list of topics
 * Steps:
 
+## Specific Configuration Options
+
+| Name | Type | Default Value | Description |
+|---------|----------|----------|----------|
+| load-op-timeoutMillis | integer | 300000 | The event read and create timeout in milliseconds |
+| storage-driver-create-key-enabled | boolean | false | Creates a record with or without a key |
+| storage-net-sndBuf | integer | 131072 | The size of the TCP send buffer to use when sending data. If the value is -1, the OS default will be used. |
+| storage-net-rcvBuf | integer | 32768 | The size of the TCP receive buffer to use when reading data. If the value is -1, the OS default will be used. |
+| storage-driver-request-size | integer | 1048576 | The maximum size of a request in bytes. This setting will limit the number of record batches the producer will send in a single request to avoid sending huge requests. |
+| storage-driver-batch-size | integer | 16384 | Allows you to change the batch size |
+| storage-net-linger | integer | 0 | The delay before sending the records. This setting gives the upper bound on the delay for batching: once we get *batch.size* worth of records for a partition it will be sent immediately regardless of this setting, however if we have fewer than this many bytes accumulated for this partition we will 'linger' for the specified time waiting for more records to show up. |
+| storage-driver-buffer-memory | long | 33554432 | The total bytes of memory the producer can use to buffer records waiting to be sent to the server. |
+| storage-driver-compression-type | string | none | The compression type for all data generated by the producer. The default is none (i.e. no compression). Valid values are none, gzip, snappy, lz4, or zstd.  |
```

Comment:
let it be the default value to be `null` (meaning "none")
also, the type is enum rather than string